plugins/available/bash-my-aws.plugin.bash

```diff
@@ -0,0 +1,44 @@
+cite about-plugin
+about-plugin 'Bash My AWS'
+
+export BASH_MY_AWS="${BASH_MY_AWS:-$HOME/.bash-my-aws}"
+
+__bma_load() {
+  local force=$1
```

Comment:
This function appears to reference positional arguments that are never given. When running shellcheck:
```
__bma_load references arguments, but none are ever passed.
```
If this method does in fact need to reference the first argument `$1` then it should be invoked with a positional argument `__bma_load() "foo"`